app/views/employees/submitted_forms.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="container mt-5">
+  <div class="row mt-4">
+    <div class="col-4">
+      <div class="card" style="width: 18rem;">
+        <div class="card-body">
+          <h5 class="card-title">Working</h5>
+          <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's
+            content.</p>
+          <%= link_to "Working",employee_path(current_user.id, {_type: "working", form_id: 2}), class: "btn btn-primary" %>
```

Comment:
use proper spacing and indentation also why we have hardcoded value form_id: 2 here?

Reply:
form_id 2 for getting working form, from submissions